CONTRIBUTING.md

```diff
@@ -13,11 +13,9 @@ updated.
 Tests can be run with command:
 
 ```bash
-npm run test
+npm test
 ```
 
 ## Release
 
-* Commit all changes
-* Run `./node_modules/.bin/releasor --bump minor`, which will create new tag and publish code to GitHub and npm. See https://github.com/kimmobrunfeldt/releasor for options
-* Edit GitHub release notes
+Use [np](https://www.npmjs.com/package/np) to create a new release
```

Comment:
That's fine -- I've been doing manual `npm version <major|minor|patch>` + `git push --tags origin HEAD` as an alternative 😃

Reply:
Yeah either way is good :) I use np just to remember which the steps were again